src/lemlib/movements/purepursuit.cpp

```diff
@@ -101,22 +102,22 @@ std::pair<int, int> PurePursuit::update(Pose pose) {
         Pose intersect = circleLineIntersect(last, cur, pose, lookaheadDist);
         // if the intersection is not the robot's current position, then we have found the lookahead point
         if (intersect != pose) {
-            lookahead = {intersect.x, intersect.y, 0, 0, i};
+            lookahead = {intersect.x, intersect.y, 0_deg, 0_rpm, i};
             break;
         }
     }
     prevLookahead = lookahead; // update previous lookahead position
 
     // get the curvature of the arc between the robot and the lookahead point
-    float curvature = getCurvature(pose, lookahead);
+    Curvature curvature = getCurvature(pose, lookahead);
 
     // get the target velocity of the robot
-    float targetVel = closest.speed;
+    AngularVelocity targetVel = closest.speed;
     // calculate target left and right velocities
-    float leftVel = targetVel * (2 + curvature * trackWidth) / 2;
-    float rightVel = targetVel * (2 - curvature * trackWidth) / 2;
+    AngularVelocity leftVel = targetVel * (Number(2) + curvature * trackWidth) * 0.5;
+    AngularVelocity rightVel = targetVel * (Number(2) - curvature * trackWidth) * 0.5; // todo: test
```

Comment:
```suggestion
    LinearVelocity leftVel = targetVel * (Number(2) + curvature * trackWidth) * 0.5;
    LinearVelocity rightVel = targetVel * (Number(2) - curvature * trackWidth) * 0.5; // todo: test
```
Change to correct units